renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ]
```

Comment:
```suggestion
  ],
  "packageRules": [
    // deno_core updates require manual adjustments
    {
      "matchPackageNames": ["deno_core"],
      "matchManagers": ["cargo"],
      "enabled": false
    }
  ]
```

Reply:
Rename this file to `renovate.json5` and it'll still work.

Reply:
Ok, I updated this file a couple times and brought in what I think is a good configuration:

- Ignore `/federation-1/**`
- Group npm >= 1.x together
- Group cargo >= 1.x together
- Majors are all separate
- No auto-merge for now.

If you expand the summary of this description, it'll show you what it'll do now.  Looks like 19 PRs.  Probably not a bad idea to get started trying to land some of those this week.